unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/BuilderMode/BIWMainController.cs

```diff
@@ -732,6 +733,7 @@ public void ExitEditMode()
 
         biwAudioHandler.gameObject.SetActive(false);
         DataStore.i.appMode.Set(AppMode.DEFAULT);
+        DataStore.i.virtualAudioMixer.sceneSFXVolume.Set(1f);
```

Comment:
This shouldn't have into account the last value? I mean, if somewhere else put a value of 0,5f this will override it

Reply:
No, this `sceneSFXVolume` is not the value taken by the settings panel but a independent multiplier that is applied to it. If the value in Settings Panel was 0.5, the fact to put this multiplier as 0 and then to 1 will will always keep the original value.

Reply:
Sounds good!